<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выйти из учётной записи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="birdBreedButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Порода птицы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="customerButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Заказчик</string>
    </property>
   </widget>
   <widget class="QPushButton" name="feedingButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кормёжка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="foodButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Корм</string>
    </property>
   </widget>
   <widget class="QPushButton" name="poultryHouseButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Птичник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="providerButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поставщик</string>
    </property>
   </widget>
   <widget class="QPushButton" name="provisionButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поставка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="workerButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сотрудник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="workerTeamButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Бригада</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-20</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите таблицу</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
